examples/tensorflow/classification/main.py

```diff
@@ -205,12 +205,16 @@ def run(config):
             compression_ctrl, compress_model = create_compressed_model(model, nncf_config, compression_state)
             compression_callbacks = create_compression_callbacks(compression_ctrl, log_dir=config.log_dir)
 
-            scheduler = build_scheduler(
-                config=config,
-                steps_per_epoch=train_steps)
-            optimizer = build_optimizer(
-                config=config,
-                scheduler=scheduler)
+            train_kwargs = {}
+
+            if 'train' in config.mode:
+                scheduler = build_scheduler(
```

Comment:
It looks like the following change is enough and does not need to change the main.py
```
base_lr = schedule_params.get('base_lr', optimizer_config.get('base_lr', 0.001))
```

Reply:
Done